cloud/openstack/os_server_facts.py

```diff
@@ -64,10 +77,16 @@ def main():
 
     try:
         cloud = shade.openstack_cloud(**module.params)
-        server = cloud.get_server(module.params['server'])
-        hostvars = dict(openstack=meta.get_hostvars_from_server(
-            cloud, server))
```

Comment:
I guess it is changing the default, but this module has not yet seen a release, so now seems the right time to be doing this.  My reasoning is that it is better not to impose the load ot the additional API calls without explicitly requesitng that behavior, but if folks feel strongly we can default detailed to True.